packages/core/package.json

```diff
@@ -111,7 +111,7 @@
     "pirates": "^4.0.1",
     "pluralize": "^8.0.0",
     "prettier": "^2.5.0",
-    "prisma": "3.6.0",
+    "prisma": "3.9.0",
```

Comment:
Are we on a non-caret version for any particular?

Reply:
We rely on APIs to do migrations that break within non major versions